scanner/src/test/kotlin/ScannerWrapperConfigTest.kt

```diff
@@ -59,5 +60,24 @@ class ScannerWrapperConfigTest : WordSpec({
 
             ScannerWrapperConfig.create(options).second shouldContainExactly mapOf("other" to "value")
         }
+
+        "set unset storage options to null" {
```

Comment:
I don't understand the test case name. Mind giving a hint?

Reply:
I guess you should read it like "storage option that are unset in the config should be explicitly set to null in the class properties".

Reply:
Is the new name better?